battles/helpers.py

```diff
@@ -0,0 +1,16 @@
+from .models import BattleTeam
+
+
+def can_run_battle(battle):
+    creator_team = BattleTeam.objects.filter(
+        battle_related=battle, trainer=battle.creator).exists()
+    opponent_team = BattleTeam.objects.filter(
+        battle_related=battle, trainer=battle.opponent).exists()
+    return creator_team and opponent_team
+
+
+def check_and_run_battle(battle_id):
```

Comment:
You are passing the whole object here, not the id

Reply:
Oops. Thanks!